core_backend/requirements.txt

```
@@ -19,4 +19,5 @@ types-openpyxl==3.1.4.20240621
 redis==5.0.8
 python-dateutil==2.8.2
 gTTS==2.5.1
-google-cloud-storage==2.18.2
+backoff==2.2.1
+google-cloud-storage==2.18.2
```

There should be a new line at the end of every file. This should be caught be pre-commit hooks!

```suggestion
google-cloud-storage==2.18.2

```